ios/Handlers/MobileAuthHandler.swift

```diff
@@ -32,12 +25,13 @@ class MobileAuthHandler: NSObject {
 }
 
 extension MobileAuthHandler : MobileAuthConnectorToHandlerProtocol {
-    func enrollForMobileAuth(_ completion: @escaping (Bool?, Error?) -> Void) {
+    
+    func enrollForMobileAuth(_ completion: @escaping (Error?) -> Void) {
         ONGClient.sharedInstance().userClient.enroll { enrolled, error in
             if let error = error {
-                completion(false, error);
+                completion(error);
               } else {
-                completion(true, nil)
+                completion(nil)
               }
```

Comment:
If `error` is optional here and you don't use `enrolled` you can just call `completion(error)` instead of the whole `if` statement :)